spec/metadata-1.1.html

```diff
@@ -93,26 +93,34 @@ <h2>FPCore metadata properties</h2>
     synonymous with such a list.
 
     <dl class="code-terms">
-      <dt>(float <var>e</var> <var>s</var>)</dt>
+      <dt>(float <var>e</var> <var>nbits</var>)</dt>
       <dd>An IEEE-754 binary floating-point value with <var>e</var>
-      exponent bits and <var>s</var> significant bits (including the
-      implicit bits). <code>binary64</code> and <code>binary32</code>
-      are common shorthands.</dd>
+      exponent bits and <var>nbits</var> total bits.
+      There will be <var>nbits</var> - <var>e</var> significand bits
```

Comment:
What about the sign bit? And how exactly are we counting implicit bits? I think it would help to give the exact values of the shorthands. I think they are: `binary32` is `(float 8 32)` and `binary64` is `(float 11 64)`, but I could see how you'd read this and think either of those numbers should be one higher or lower.